e2e/cypress/integration/polis/client-admin/create_user.spec.js

```diff
@@ -1,45 +1,30 @@
 describe('Create User', () => {
-  afterEach(() => cy.visit('/signout'))
-
   beforeEach(() => {
-    cy.visit('/signout')
-    cy.visit('/createuser')
+    // Cypress doesn't believe in cleanup.
+    // See: https://docs.cypress.io/guides/references/best-practices.html#State-reset-should-go-before-each-test
+    cy.logout()
   })
 
-  before(async function () {
-    const users = await cy.fixture('users')
-    this.user = users[0]
+  before(() => {
+    cy.fixture('users.json').as('users')
   })
 
-  it('Does not create a new user with existing email address', () => {
-    // Create user.
-    cy.get('form').within(function () {
-      cy.get('input[placeholder=name]').type(this.user.name)
-      cy.get('input[placeholder=email]').type(this.user.email)
-      cy.get('input[placeholder=password]').type(this.user.password)
-      cy.get('input[placeholder="repeat password"]').type(this.user.password)
+  it('Does not create a new user with existing email address', function () {
+    const user = this.users[0]
 
-      cy.get('button').click()
-    })
+    // Create user.
+    cy.signup(user.name, user.email, user.password)
 
     // Attempt to recreate user.
-    cy.visit('/createuser')
-    cy.get('form').within(function () {
-      cy.get('input[placeholder=name]').type(this.user.name)
-      cy.get('input[placeholder=email]').type(this.user.email)
-      cy.get('input[placeholder=password]').type(this.user.password)
-      cy.get('input[placeholder="repeat password"]').type(this.user.password)
-
-      cy.get('button').click()
-    })
+    cy.signup(user.name, user.email, user.password)
 
     cy.get('form').contains(
       'Email address already in use, Try logging in instead.'
     )
     cy.url().should('eq', Cypress.config().baseUrl + '/createuser')
   })
 
-  it('Creates a new user', () => {
```

Comment:
Surprisingly to me, this change is needed in order for the test context in `this` to be available..! e.g., fixtures from `before()`, etc.

Reply:
yes. This is the difference between classical function and arrow function. The arrow function does not rebind `this`.

Reply:
yay i'm lerning 😄